### PR TITLE
#241: Implementation PFMERGE command

### PR DESCRIPTION
**Pull request by lucifercr07**
- Adding support for [PFMERGE](https://redis.io/docs/latest/commands/pfmerge/) command implementation.
- Added unit/integration tests.
- TODO:
  - The switch to `DENSE` representation for Hyperloglog would be added once #446 is resolved.

### CODE REVIEW

**Review by JyotinderSingh — CHANGES_REQUESTED** [review]
Thanks for adding support for the `PFMERGE` command @lucifercr07!

Left a few comments.

File: internal/eval/eval.go [hunk 1]
```diff
@@ -2603,3 +2603,41 @@ func evalPFCOUNT(args []string, store *dstore.Store) []byte {
...
+		mergedHll, ok = obj.Value.(*hyperloglog.Sketch)
+		if !ok {
+			return diceerrors.NewErrWithMessage(diceerrors.WrongTypeHllErr)
+		}
+	}
+
+	for _, arg := range args {
+		obj := store.Get(arg)
+		if obj != nil {
+			currKeyHll := obj.Value.(*hyperloglog.Sketch)
```
Comment (JyotinderSingh) [inline internal/eval/eval.go:2630]: We should be doing a type assertion here as well

**Review by JyotinderSingh — CHANGES_REQUESTED** [review]

File: internal/eval/eval.go (hunk 1 above)
Comment (lucifercr07) [reply internal/eval/eval.go]: My bad missed it 🤦‍♂️ . Thanks for review.

**Review by JyotinderSingh — APPROVED** [review]
Thanks for adding support for `PFMERGE`, @lucifercr07! LGTM.

Please resolve the conflicts.

**Pull request merged**